framework/branch-key-store.md

```diff
@@ -478,6 +639,10 @@ TransactWriteItemRequest:
 If DDB TransactWriteItems is successful, this operation MUST return a successful response containing no additional data.
 Otherwise, this operation MUST yield an error.
 
+The condition expression for the Active Input ensures
+the Active Item in storage has not changed since it was read.
+This prevents overwrites due to a race in updating the Active Item.
```

Comment:
@rishav-karanjit these lines need to be added to https://github.com/awslabs/aws-encryption-sdk-specification/pull/297